blobfuse2-release.yaml

```diff
@@ -104,7 +104,6 @@ stages:
               cp azure-storage-fuse/setup/baseConfig.yaml pkgDir/usr/share/blobfuse2/
               cp azure-storage-fuse/sampleFileCacheConfig.yaml pkgDir/usr/share/blobfuse2/
               cp azure-storage-fuse/sampleStreamingConfig.yaml pkgDir/usr/share/blobfuse2/
-              cp azure-storage-fuse/tools/postinstall.sh pkgDir/usr/share/blobfuse2/
```

Comment:
Why did we remove this?

Reply:
postinstall is basically installing the bash scripts for auto-completion. For fstab to work, we had to make mount as a default command option which creates ambiguity in commands like "blobfuse2 abc" = "blobfuse mount abc" and hence autocompletion scripts are failing due to this. Untill we find a way to fix those we are removing these scripts, fstab being a must have feature and auto-completion can be ignored.